cli.js

```diff
@@ -7,66 +7,90 @@ const teti = require('./');
 const ora = require('ora');
 const chalk = require('chalk');
 const table = require('text-table');
-const fkill = require('fkill');
-const psList = require('ps-list');
 
 const cli = meow(`
   Usage
     $ teti <url>
 
   Options
-    -n          number of tests to run (10 is default)
-    --verbose   output all data
+    --number, -n    number of tests to run (10 is default)
+    --runner, -r    specify runner (chrome is default)
+    --custom, -c    additional metrics to gather (Navigation Timing API and User Timing API)
+    --output, -o    output format: table or csv, (table is default)
+    --verbose, -v   output all data
 
   Examples
     $ teti google.com -n 96
-`);
+`, {
+    flags: {
+        number: {
+            type: 'number',
+            alias: 'n',
+            default: 10
+        },
+        runner: {
+            type: 'string',
+            alias: 'r',
+            default: 'chrome'
```

Comment:
HeadlessChrome?

Reply:
In my opinion, there is no need for that. At the time we support only one runner - `chrome`. Testing "with a head" doesn't make much sense (unless debugging?) and the other provider that may be added eventually would be `firefox`. If we really need a switch for headless browsing, I would recommend orthogonal flag, like simply `--headless`